lib/dw/chart.js

```diff
@@ -189,8 +192,13 @@ export default function(attributes) {
             return attributes;
         },
 
-        onChange: changeCallbacks.add,
+        // Super-simple event handling, borrowed from Svelte 2:
+        _handlers: {},
```

Comment:
what does `_handlers` do?

Reply:
It's the object where event handlers get stored in Svelte 2.

Reply:
(used by `on` and `fire`)